src/main/java/teammates/common/datatransfer/CommentAttributes.java

```diff
@@ -328,9 +328,13 @@ public String getEditedAtText(Boolean isGiverAnonymous) {
             String displayTimeAs = TimeHelper.formatDateTimeForComments(this.lastEditedAt);
             return "(last edited " +
                     (isGiverAnonymous ? "" : "by " + this.lastEditorEmail + " ") +
-                    "at " + displayTimeAs + ")";
+                    "at <span class=\"client-time\">" + displayTimeAs + "</span>)";
         } else {
             return "";
         }
     }
+
+    public String getCreatedAtText() {
```

Comment:
Leave it as it is to keep in consistent. But create a new issue to refactor it, assign it to yourself, and do after this one is done.